src/app/public/src/modules/open-on-click-directive/open-on-click.directive.ts

```diff
@@ -0,0 +1,40 @@
+import { Directive, HostListener, Input, Renderer2, ElementRef } from '@angular/core';
+
+import { HelpWidgetService } from '../shared';
+
+@Directive({
+  selector: '[bbHelpOpenOnClick]'
+})
+export class BBHelpOpenOnClickDirective {
+  @Input()
+  public bbHelpOpenOnClick: string;
```

Comment:
Curious if we threw the word "Key" in there to indicate that this input accepts a help key? (If that's the intent, of course)
```
<button bbHelpOpenKeyOnClick="default.htm">
```

Reply:
The helpKey being passed in is optional, without the key, it will still trigger the widget to open to the currentHelpKey being used by the widget.  I don't think making the directive name longer with `Key` is necessary, or accurate in all cases.  The general concept is just, add this and you can click (or hit enter) on the element to open the widget.